src/js/experimental/media-chrome-selectmenu.js

```diff
@@ -115,17 +115,17 @@ class MediaChromeSelectMenu extends window.HTMLElement {
   }
 
   #handleChange_() {
-    this.#toggle();
+    this.#toggle(true);
```

Comment:
if we get a change event, we just want to close the selectmenu as a new item has been selected. We don't want to accidentally open it if it's been programmatically set.